website/docs/cli.md

```diff
@@ -3,8 +3,83 @@ id: cli
 title: CLI
 ---
 
-TODO: Add CLI reference here.
+Docusaurus provides a set of scripts to help you generate, serve, and deploy your website. 
 
-#### References
+Once your website is generated, your website package will contain the Docusaurus scripts that you may invoke with your package manager:
 
-- TODO
+```json
+// package.json
+{
+  // ...
+  "scripts": {
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy"
+  }
+}
+```
+
+## Docusaurus CLI commands
+
+Below is a list of Docusaurus CLI commands and their usages:
+
+### `docusaurus init`
```

Comment:
Can we omit this part first ?

And also it's a misnomer to use `docusaurus` because v2 will be using scoped package
E.g: `@docusaurus/core` or `@docusaurus/cli` later on

Reply:
Updated 🦊